packages/site/src/runtimeTypeChecking.mdx

```diff
@@ -263,6 +263,11 @@ const sumModelType = types.refinement(types.model<Sum>(Sum), sum => {
 })
 ```
 
+### Syntax sugar for optional primitives with a default value
+
+You can also do `tProp(defaultValue: string | number | boolean)`, which is equivalent to `tProp(types.string|number|boolean, defaultValue)`.
+In other words, if you use `tProp(42)`, then the property will be a number that takes the value `42` when undefined.
```

Comment:
I would use something like `then the property will be a number and use `42` as a default if the value is `undefined`, because it got me confused, I've got it only after reading it for 3rd time or so.

Reply:
thanks! clarified